var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -45,19 +47,45 @@ class PyYt(PythonPackage):
     variant("scipy", default=True, description="enable scipy support")
     variant("rockstar", default=False, description="enable rockstar support")
```

Comment:
Since you're the new maintainer of this package... I don't love the way variants are done here. I usually try to stick to the same variants as pip. There's no `pip install yt[astropy]`, there's only `pip install yt[full]`. I would personally make a `+full` variant and default to false to minimize the build. But it's up to you now.

Reply:
Then I will leave it as is.